nototools/docker/make_getnoto/build_noto_site_data.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Build data for get/noto site.
+
+# Map /tmp/website2 to an external location to get the data when this is
+# complete.
+
+# Put the repos at the latest tagged release.  This does not do a git pull, we
+# assume noto is already at or beyond the latest release.  Might want to rethink
+# that.
+cd /app/noto/nototools/nototools
+./sync_repos.py -v
+
+# Add missing font package to the CJK repo.
+# This is too big for github.  Currently we fetch it from the website itself,
+# which is kind of circular, since we're building the website...  We need a 
+# 'prerelease' location for things like this.
```

Comment:
Yeah, this comment is not so clear.  This is actually fetching the zip from gcs mapped through our website, and putting it into the cjk dir.  Then we process the cjk dir files, including this, and write to our local staging directory.  From there it will later get pushed to gcs (not in this script).

This is using public urls so I don't need to pass any authentication info through to the script (wasn't sure how to best set that up-- of course I should figure that out).